package.json

```diff
@@ -27,11 +27,12 @@
   "dependencies": {
     "@actions/core": "^1.2.5",
     "collections": "^5.1.12",
+    "glob": "^7.1.6",
```

Comment:
`glob` is currently a transitive dependency, but since we use it directly, I thought it might be better to define it.